doc/references.bib

```diff
@@ -100,3 +99,36 @@ @article{NolteEtAl2004
   volume = {115},
   year = {2004}
 }
+@article{Hosking1980,
+  author = {J. R. M. Hosking},
+  issn = {01621459},
+  journal = {Journal of the American Statistical Association},
+  number = {371},
+  pages = {602--608},
+  publisher = {[American Statistical Association, Taylor & Francis, Ltd.]},
+  title = {The Multivariate Portmanteau Statistic},
+  url = {http://www.jstor.org/stable/2287656},
+  volume = {75},
+  year = {1980}
+}
+
+@article{kilian_2006,
+  author = {Kilian, Lutz},
+  doi = {10.1017/S0266466606000442},
+  journal = {Econometric Theory},
+  number = {5},
+  pages = {961–967},
+  publisher = {Cambridge University Press},
+  title = {NEW INTRODUCTION TO MULTIPLE TIME SERIES ANALYSIS, by Helmut Lütkepohl, Springer, 2005},
+  volume = {22},
+  year = {2006},
+}
+
+@article{li_linear_2017,
+	author = {Li, A. and Gunnarsdottir, K.M. and Inati, S. and Zaghloul, K. and Gale, J. and Bulacio, J. and Martinez-Gonzalez, J. and Sarma, S.V.},
+	doi = {10.1109/EMBC.2017.8037439},
+	isbn = {978-1-5090-2809-2},
+  journal = {IEEE, EMBC},
+	title = {Linear time-varying model characterizes invasive {EEG} signals generated from complex epileptic networks},
+	year = {2017},
+}
```

Comment:
same here. Use something like this as suggested by IEEE page: https://ieeexplore.ieee.org/abstract/document/8037439

@INPROCEEDINGS{li_linear_2017,
  author={Li, Adam and Gunnarsdottir, Kristin M. and Inati, Sara and Zaghloul, Kareem and Gale, John and Bulacio, Juan and Martinez-Gonzalez, Jorge and Sarma, Sridevi V.},
  booktitle={2017 39th Annual International Conference of the IEEE Engineering in Medicine and Biology Society (EMBC)}, 
  title={Linear time-varying model characterizes invasive EEG signals generated from complex epileptic networks}, 
  year={2017},
  volume={},
  number={},
  pages={2802-2805},
  doi={10.1109/EMBC.2017.8037439}}

Reply:
Fixed.